manifest.md

```diff
@@ -92,7 +92,7 @@ A client will distinguish a manifest list from an image manifest based on the Co
 ## Example Manifest List
 
 *Example showing a simple manifest list pointing to image manifests for two platforms:*
-```json
+```application/vnd.oci.image.manifest.list.v1+json
```

Comment:
It could be argued that github's syntax highlighting is broken... this just seemed very natural. Let's go with this until it breaks down.